src/network/networkprotocol.h

```diff
@@ -219,7 +219,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define PASSWORD_SIZE 28       // Maximum password length. Allows for
                                // base64-encoded SHA-1 (27+\0).
 
-#define FORMSPEC_API_VERSION 1
+#define FORMSPEC_API_VERSION 2
```

Comment:
Don't increment this, this might cause very erroneous things to happen to tabheaders on older clients.  It may work alright, but it's better to add things to `style` now.